subsys/bluetooth/controller/ll_sw/ull_conn_iso.c

```diff
@@ -15,6 +15,10 @@
 #include "hal/ccm.h"
 #include "hal/ticker.h"
 
+#if defined(CONFIG_BT_LL_SW_LLCP)
+#include "ull_tx_queue.h"
+#endif /* CONFIG_BT_LL_SW_LLCP */
```

Comment:
Why is the include being guarded?

Reply:
This include is only required for the new LLCP, and not for the legacy LLCP

Reply:
Fair, but it should be fine to include unconditionally, right?

Reply:
That would work, but we have put the guard around at all other locations we needed this. 
I suggest to leave the guard in place, and remove the guard when removing the legacy LLCP